docs/tutorials/odc-stac.rst

```diff
@@ -0,0 +1,290 @@
+
+Load items with odc-stac
+------------------------
+
+After producing a list of items to load, we can use the :code:`load` function from :code:`odc-stac` to read the requested assets from the items and return them as xarrays.
+
+Type the following into the empty cell below the **Load items with odc-stac** heading:
+
+.. code-block:: python
+
+   ds = load(
+      items=items,
+      bands=assets_query,
+      geopolygon=aoi_geometry,
+      crs="utm",
+      resolution=30
+   )
+
+   ds
+
+When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
+You may see warnings appear in red text.
+This is expected behaviour for this dataset and does not mean anything is wrong.
+After running the cell, you should see a :code:`xarray.Dataset` summary.
+
+.. image:: ../_static/tutorial-images/odc-stac/xarray_output.png
+ :width: 600
+ :alt: A table showing a summary of the loaded xarray, including dimensions, coordinates, and data variables.
+ :align: center
+
+Visualise loaded data
+---------------------
+
+To confirm that we have loaded the requested data, it is useful to visualise it. 
+We can use :code:`xarray`'s built-in plotting functionality to make a basic plot.
+
+Type the following into the empty cell below the **Visualise loaded data** heading:
+
+.. code-block:: python
+
+   ds.data.plot.imshow(col="time", col_wrap=3)
+
+When you have finished, run the cell by pressing :code:`Shift+Enter` on your keyboard.
+After running the cell, you should see the following visualisation.
+
+.. image:: ../_static/tutorial-images/odc-stac/output.png
+ :width: 600
+ :alt: A series of images showing the land cover data for each year, from 2017 to 2023. Each image shows the island in green and yellow, surrounded by ocean in blue.
+ :align: center
+
+Advanced visualisation
+^^^^^^^^^^^^^^^^^^^^^^
+
+The code we've used provides us a visualisation that allows us to check that the data loaded successfully.
+
+To produce a more descriptive plot, we recommend reviewing the `example notebook <https://planetarycomputer.microsoft.com/dataset/io-lulc-annual-v02#Example-Notebook>`_ that Microsoft Planetary Computer provide for this dataset.
+
+Export loaded data
+------------------
+
+Once you have loaded and checked your data, it is often useful to export it.
+This allows you to use the data in other software and analyses.
+The :code:`odc-geo` package adds the `.odc` extension for `xarray` datasets, allowing easy exporting of Cloud Optimised GeoTIFFs.
+
+The code below extracts the year of each image from the dataset, then uses a loop to export each dataset to a new file.
+
+Type the following into the empty cell below the **Export loaded data** heading:
+
+.. code-block:: python
+
+   years = ds.time.dt.strftime("%Y").values
+
+   for timestep in range(len(ds.time)):
+   
+       ds_single_year = ds.data.isel(time=timestep)
+       
+       write_cog(
```

Comment:
I think we should use the `.odc.write_cog()` extension instead of importing the function itself. It's a really nice way to interact with ODC data, and once users are familiar with it there's a lot of handy tools there they can use.

Would also fit with the sentence above: `adds the `.odc` extension for `xarray` datasets, allowing easy exporting of Cloud Optimised GeoTIFFs.`

Reply:
I think the text is a hang-over of when I was using the .odc extension, which I started doing because I also liked it, but then worried that it might be one thing too many to explain to the user. 

What would you think of us writing another tutorial to highlight the useful functionality of odc-geo and the .odc extension?